.github/workflows/nessie.yaml

```diff
@@ -724,6 +727,7 @@ jobs:
           LAKEFS_BLOCKSTORE_TYPE: azure
           LAKEFS_BLOCKSTORE_AZURE_STORAGE_ACCOUNT: ${{ secrets.LAKEFS_BLOCKSTORE_AZURE_STORAGE_ACCOUNT }}
           LAKEFS_BLOCKSTORE_AZURE_STORAGE_ACCESS_KEY: ${{ secrets.LAKEFS_BLOCKSTORE_AZURE_STORAGE_ACCESS_KEY }}
+          NESSIE_BLOCKSTORE_TYPE: azure
```

Comment:
Can this be different than `LAKEFS_BLOCKSTORE_TYPE`? Why not use the same env var?

Reply:
It is used inside `nessiecontext, and `viper` only reads variables with `NESSIE_` prefix